tests/Microsoft.Identity.Test.Unit.net45/PublicApiTests/ConfidentialClientApplicationTests.cs

```diff
@@ -399,7 +421,91 @@ await app
                       .WithForceRefresh(true)
                       .ExecuteAsync(CancellationToken.None)
                       .ConfigureAwait(false);
-                Assert.AreNotEqual(cachedAssertion, app.ClientCredential.Assertion);
+                Assert.AreNotEqual(cachedAssertion, app.ClientCredential.CachedAssertion);
```

Comment:
Will the content be the same though? Perhaps that should be validated as well?

Reply:
The content will be the same, this is simply a name change to signify the difference between a user provided assertion and an assertion that we create and cache from signing operations